### ISSUE TITLE
Version 3.11 - no login possible in Homeassistant

### ISSUE THREAD

**Issue by DavidWelsch** [GimpArm/smartfriends-bridge#6]
Hi,
when I updated to 3.11 the Addon stopped working in Homeassistant.
I get the following Exception in the logs
`info: SmartFriends.Api.Session[0]
      Connecting to 10.1.4.142
fail: SmartFriends.Api.Session[0]
      Failed to open socket to host
      System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception.
       ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL.
       ---> Interop+Crypto+OpenSslCryptographicException: error:0A0000B5:SSL routines::no ciphers available
         --- End of inner exception stack trace ---
         at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, ReadOnlySpan`1 input, Byte[]& sendBuf, Int32& sendCount)
         at System.Net.Security.SslStreamPal.HandshakeInternal(SafeDeleteSslContext& context, ReadOnlySpan`1 inputBuffer, Byte[]& outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
         --- End of inner exception stack trace ---
         at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](Boolean receiveFirst, Byte[] reAuthenticationData, CancellationToken cancellationToken)
         at SmartFriends.Api.Client.Open() in D:\Projects\smartfriends-bridge\source\SmartFriends.Api\Client.cs:line 70`

I didn't change my config, just updated the addon.

As I see it, the authentication fails. It worked before the update, so I dont think its a problem on the Bridge.

Please let me know if you need additonal information.
Thanks in advance.
David

**Comment by GimpArm** [comment]
I know this was already reported in the add-on repo https://github.com/GimpArm/hassio-addons/issues/7

I'm sorry, in my rush to push it out, the ARM64 version was seg faulting on the docker build due to a Libc issue unrelated to anything I changed. Updating from debian bullseye to bookworm fixed the seg fault problem and was the suggested solution, but libssl1.1 was no longer available and I had to use libssl2. and I pushed it without testing because I was rushing.

Anyways, downgrade to version 3.10 for now. I pulled the tag so no one else on ARM64 can upgrade. I wont be able to build and test on an ARM64 until Friday, so I'll hopefully get a fix for that.
